napari/_qt/qt_main_window.py

```diff
@@ -146,8 +152,14 @@ def _add_view_menu(self):
         toggle_visible.setShortcut('Ctrl+M')
         toggle_visible.setStatusTip('Hide Menubar')
         toggle_visible.triggered.connect(self._toggle_menubar_visible)
+        dark_mode = QAction("Dark mode", self._qt_window)
```

Comment:
I think I'd rather see the `QAction` be called `Toggle theme`, with the status tip `Toggle theme` and the keybinding `Control+Shift+T` as is used https://github.com/napari/napari/blob/c01024f151546f7a2ada9727572efb3e65fa4f0f/napari/keybindings.py#L15. There shouldn't be any special references to light / dark here

Once this is done you can remove the code here https://github.com/napari/napari/blob/c01024f151546f7a2ada9727572efb3e65fa4f0f/napari/keybindings.py#L15

Reply:
I renamed this and removed the keybinding. The keybinding no longer shows up in the "keybindings help" window, but I guess that isn't an issue since it is visible in the main menu. 

I changed `Ctrl-Shift-O` to `Ctrl+Shift+O` for the open folder shortcut so that Qt recognizes this and displays the shortcut it in the menu.